agreements/project-contribution/asset-transfer-agreement.md

```diff
@@ -0,0 +1,216 @@
+# Asset Transfer Agreement
+
+**Effective Date:** `__________________________`  
+
+---
+
+**Project:** `_________________________________` (“Project”)  
+
+*Assignor(s):*
+
+- `__________________________________________________________`  
+- `__________________________________________________________`  
+- `__________________________________________________________`  
+
+*Principal Office or Primary Address of Assignor(s):*
```

Comment:
I only have my personal address to use. I don't want that publicly visible. These documents are private, right?

Reply:
100%.